google/samus/modules/cmt.nix

```diff
@@ -0,0 +1,23 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.hardware.samus.cmt;
+in
+
+{
+  options.hardware.samus.cmt = {
```

Comment:
This rather belongs to `services.xserver.cmt` by current convention.

Reply:
Ok. I'll do so.